cli/npm/registry.rs

```diff
@@ -100,6 +100,8 @@ pub struct NpmPeerDependencyMeta {
 pub struct NpmPackageVersionInfo {
   pub version: String,
   pub dist: NpmPackageVersionDistInfo,
+  #[serde(default)]
+  pub bin: HashMap<String, String>,
```

Comment:
https://registry.npmjs.org/cowsay - Tests will follow in the `deno task` running npm scripts PR.

Reply:
Looks like some packages just have a string here.